files/usr/lib/systemd/system/system-flatpak-setup.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Manage system flatpaks
+Documentation=https://github.com/ublue-os/endlish-oesque/issues/10
```

Comment:
This `Documentation=https://github.com/ublue-os/endlish-oesque/issues/10` seems outdated / not applicable, no?

Reply:
Ah, yeah, that's a leftover from bringing the files in from bazzite. Removing it now